### ISSUE TITLE
Url parameters for fanout settings

### ISSUE THREAD

**Issue by mrpatpat** [MaibornWolff/codecharta#156]
Original issue was #119. Depends on #151 #152 #153 #154

## Description

All the fanout settings should get own url parameters. The default case should be an automatic highlighting (#154 ) on the root node.

**Comment by ukinimod** [comment]
I don't really think such parameters are needed, see #158.

**Comment by Richargh** [comment]
This seems no longer relevant.
